### PR TITLE
Adding semsimian package from github source

### PR DESCRIPTION
**Pull request by dillonroach**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by h-vetinari** [comment]
I'm testing a fix for osx in https://github.com/conda-forge/staged-recipes/pull/26012 and it's looking good, because I'm getting
```
+ echo 'Building all recipes'
+ python .ci_support/build_all.py
Building all recipes
Overriding MACOSX_DEPLOYMENT_TARGET to be  10.12
```
which is what we need.

**Comment by dillonroach** [comment]
@conda-forge/help-python, ready for review!

File: recipes/semsimian/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - {{ stdlib('c') }}
+    - cargo-bundle-licenses
```
Comment (xhochy) [inline recipes/semsimian/meta.yaml:21]: You also need to call `cargo-bundle-licenses` as part of the build script

File: recipes/semsimian/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  host:
+    - python
+    - maturin >=0.14,<0.15
+    - pip
+  run:
+    - python
+    - libgcc-ng  # [linux]
+    - libcxx  # [osx]
+    - vs2015_runtime  # [win]
```
Comment (xhochy) [inline recipes/semsimian/meta.yaml:30]: These should be automatically added by the run_exports of the compilers.
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/semsimian/meta.yaml (hunk 2 above)
Comment (dillonroach) [reply recipes/semsimian/meta.yaml]: it seems the pipeline breaks w/o `vs2015_runtime  # [win]` while the other two went through fine without

Comment (xhochy) [reply recipes/semsimian/meta.yaml]: This shouldn't be the case. What was the error?

**Review by xhochy — APPROVED** [review]

**Pull request merged**